src/main/fc/runtime_config.c

```diff
@@ -23,6 +23,8 @@
 #include "fc/runtime_config.h"
 #include "io/beeper.h"
 
+const char *armingDisableFlagNames[] = { ARMING_DISBALED_FLAG_NAMES };
```

Comment:
I assume this will be eliminated by the linker if `MINIMAL_CLI` is eliminated. Should we make it explicit by moving it to a conditional section in `cli.c`? This also prevents 'accidental' references to it.